RSSReader/Services/NetworkManager.m

```diff
@@ -0,0 +1,57 @@
+//
+//  NetworkManager.m
+//  RSSReader
+//
+//  Created by Владислав Станкевич on 19.11.20.
+//
+
+#import "NetworkManager.h"
+#import "FeedItem.h"
+
+@interface NetworkManager ()
+
+@property (nonatomic, strong) NSURLSession *session;
+
+@end
+
+@implementation NetworkManager
+
+- (instancetype)initPrivate {
+    self = [super init];
+    _session = [NSURLSession sharedSession];
```

Comment:
you don't retain object and release it In dealloc, violating MRC principles
add retain